llvm/test/CodeGen/AArch64/pr-cf624b2.ll

```diff
@@ -11,45 +11,23 @@ define linkonce_odr void @_ZN1y2beEPiRK1vPmPS1_(<8 x i8> %0, ptr %agg.tmp.i) {
 ; CHECK:       // %bb.0: // %entry
 ; CHECK-NEXT:    sub sp, sp, #16
 ; CHECK-NEXT:    .cfi_def_cfa_offset 16
-; CHECK-NEXT:    mov x8, sp
 ; CHECK-NEXT:    // kill: def $d0 killed $d0 def $q0
```

Comment:
A bunch of stores are removed here, but it seems like that where of undef lanes.

Reply:
Using `NVCAST` as returned the stores (though addressing is still nicer)